com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfSensor.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace Unity.Robotics.UrdfImporter
+{
+    public class UrdfSensor: MonoBehaviour
```

Comment:
Is this class used anywhere?

Reply:
https://github.com/Unity-Technologies/URDF-Importer/blob/vidur/tf/com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfSensorExtension.cs